esp-link/cgiwifi.c

```diff
@@ -121,10 +121,11 @@ static ICACHE_FLASH_ATTR
 void wifiStartMDNS(struct ip_addr ip) {
   if (!mdns_started) {
     os_memset(&mdns_info, 0, sizeof(struct mdns_info));
-    mdns_info.host_name = flashConfig.hostname;
+    mdns_info.host_name = flashConfig.hostname,
```

Comment:
That's the possible bug fix.  All the other lines after have a comma as well so I assumed it should too.  I don't know if they were wrong or it is wrong.  The other implementations and examples I've found allocate the memory differently too so that might be the stability issue.  Here's it is mimicking other implementations:

``` c
struct mdns_info *info = (struct mdns_info *)os_zalloc(sizeof(struct mdns_info));
info->host_name = flashConfig.hostname;
info->ipAddr = ip.addr;
info->server_name = "http";
info->server_port = 80;
info->txt_data[0] = (char *) "version = now";
espconn_mdns_init(info);
```